metadata_service/proxy/atlas_proxy.py

```diff
@@ -444,46 +436,29 @@ def get_column_description(self, *,
             column_name=column_name)
         return column_detail[self.ATTRS_KEY].get('description')
 
-    @_CACHE.region('atlas_proxy', '_get_metadata_entities')
-    def _get_metadata_entities(self, popular_query_params: dict) -> List:
-        try:
-            popular_tables_guids = list()
-
-            # Fetch the metadata entities based on popularity score
-            search_results = self._driver.search_basic.create(data=popular_query_params)
-            for metadata in search_results.entities:
-                table_guid = metadata.attributes.get("table").get("guid")
-                popular_tables_guids.append(table_guid)
-
-            # In order to get comments and other extra fields from table entity
-            table_collection = self._driver.entity_bulk(guid=popular_tables_guids,
-                                                        ignoreRelationships=True)
-
-            table_entities: List = list()
-            for _collection in table_collection:
-                table_entities.extend(_collection.entities)
-
-            return table_entities
-
-        except (KeyError, TypeError) as ex:
-            LOGGER.exception(f'_get_metadata_entities Failed : {ex}')
-            raise NotFoundException('Unable to fetch popular tables. '
-                                    'Please check your configurations.')
-
     def get_popular_tables(self, *, num_entries: int) -> List[PopularTable]:
         """
         :param num_entries: Number of popular tables to fetch
         :return: A List of popular tables instances
         """
         popular_tables = list()
-        popular_query_params = {'typeName': 'table_metadata',
+        popular_query_params = {'typeName': 'Table',
                                 'sortBy': 'popularityScore',
                                 'sortOrder': 'DESCENDING',
                                 'excludeDeletedEntities': True,
-                                'limit': num_entries,
-                                'attributes': ['table']}
+                                'limit': num_entries}
+
+        table_entities: List = list()
+
+        try:
+            search_results = self._driver.search_basic.create(data=popular_query_params)
+            for table in search_results.entities:
```

Comment:
Why are we looping over entities here ? You could simply use `search_results.entities` directly on line 463.

Reply:
Agree, that try/except block seems to be redundant now. Removed this additional loop.